docs/guides/preinstall.md

```diff
@@ -4,77 +4,111 @@ These steps are common to most distros. Please refer to both this page and this
 
 ## Partition with Disk Utility
 
-In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk-utility/dskutl14027/mac). The file system format doesn't matter, but pick the amount of space that you want for Linux. You won't be able to resize your APFS partitions from a Linux installer, so you must make space now. **When prompted to, do not create an APFS volume for Linux**, you want a partition. If you want separate partitions for `/home`, `/boot` etc., create them as well.
-
-!!! note "Triple booting with Linux and Windows"
+In macOS Disk Utility you need to create your Linux partition:
+
+1. Open Disk Utility
+2. Choose "Macintosh HD *volumes*"
```

Comment:
Choose the volume you want to partition for Linux

This seems better, as the user needn't necessarily partition Macintosh HD. Also, if the user has renamed their macOS partition, then instead of Macintosh HD, the new name shall be there.

Reply:
Updated [4297a9b](https://github.com/gianmaticus/wiki/commit/4297a9bc7c5a0a832e88a59178fa8931fdbd07f3)